examples/reference/panes/Vizzu.ipynb

```diff
@@ -26,8 +26,8 @@
     "For details on other options for customizing the component see the [layout](../../how_to/layout/index.md) and [styling](../../how_to/styling/index.md) how-to guides.\n",
     "\n",
     "* **``object``** (dict | pd.DataFrame): The data expressed as a Python dictionary of arrays or DataFrame.\n",
-    "* **``animation``** (dict): Animation settings (see [vizzu.Anim](https://lib.vizzuhq.com/latest/reference/modules/Anim/)).\n",
```

Comment:
I removed these links because they where broken. Will be impossible for us to keep updated.